hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/upgrade/OMPrepareRequest.java

```diff
@@ -149,6 +157,8 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
       }
     }
 
+    auditLog(auditLogger, buildAuditMessage(OMAction.UPGRADE_PREPARE,
```

Comment:
Can we add `transactionLogIndex`  and `OMNodeId` in auditMap as an additional info for audit message ?

Reply:
TermIndex will be handled as part of "[HDDS-10658](https://issues.apache.org/jira/browse/HDDS-10658). Add Object ID and Update ID to OM audit log messages" when this is merged implicitly.
OMNodeId is not required as audit-log is per node itself. There is no need for same.